lib/rights_api/models/rights_current.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+module RightsAPI
+  class RightsCurrent < Sequel::Model(:rights_current)
+    extend ModelExtensions
+    # The obnoxious *_obj naming convention arises from the fact that apparently
+    # Sequel wants our foreign keys to be of the form attribute_id, reason_id, etc.
+    # but of course they're not, so much silliness ensues.
+    many_to_one :access_profile_obj, class: :"RightsAPI::AccessProfile", key: :access_profile
+    many_to_one :attribute_obj, class: :"RightsAPI::Attribute", key: :attr
+    many_to_one :reason_obj, class: :"RightsAPI::Reason", key: :reason
+    many_to_one :source_obj, class: :"RightsAPI::Source", key: :source
+
+    # Maybe TOO eager. This makes us partially responsible for the fact that rights_current.source
```

Comment:
`eager` and `eager_graph` kick off `JOIN`s to scarf up some portion of the tree structure of associations. If you look at the output of the `rights` or `rights_log` route, you'll see two instances of an access profile: one hanging off the main `rights` structure, and one hanging off the `rights->source` struct. I think calling just `eager` would not populate the latter. For now I think what we have is okay, since the mission is "make it return structures," I'm willing to leave this issue for a slightly later polishing step.